### PR TITLE
Catalog Type

### PR DESCRIPTION
**Pull request by abstracts33d**
image
image
image

### CODE REVIEW

File: app/controllers/catalogs_controller.rb [hunk 1]
```diff
@@ -26,12 +26,17 @@ def self.matches?(request)
...
   include ControlsCatalog
+  before_action :redirect_if_data_only
 
   def show
   end
 
   protected
 
+  def redirect_if_data_only
+    redirect_to  root_path, alert: t("catalogs.data_only", catalog_name: @catalog.name) if @catalog.data_only
```
Comment (skyporter) [inline app/controllers/catalogs_controller.rb:37]:
```suggestion
    redirect_to root_path, alert: t("catalogs.data_only", catalog_name: @catalog.name) if @catalog.data_only?
```

**Review by skyporter — APPROVED** [review]

**Review by skyporter — APPROVED** [review]

**Comment by lucaguindani** [comment]
"Custom root page" should be disabled in all views (sysadmin, catalog admin) if "Data only" is enabled.

**Comment by lucaguindani** [comment]
Data only attribute can be added to the catalog api response (https://github.com/catima/catima/blob/development/app/views/api/v3/catalogs/_catalog.json.jbuilder).

**Comment by lucaguindani** [comment]
Suggestions should not be added to the structure.sql file in this PR.

**Pull request merged**